cardano-testnet/src/Testnet/Start/Cardano.hs

```diff
@@ -138,37 +143,57 @@ getDefaultShelleyGenesis asbe maxSupply opts = do
 -- > │   │   └── vrf.{skey,vkey}
 -- > │   └── README.md
 -- > ├── drep-keys
--- > │   └── drep{1,2,3}
--- > │       └── drep.{skey,vkey}
--- > ├── genesis.{alonzo,conway}.spec.json
+-- > │   ├── drep{1,2,3}
+-- > │   │   └── drep.{skey,drep.vkey}
```

Comment:
```suggestion
-- > │   │   └── drep.{skey,vkey}
```

The `drep.{skey,vkey}` notation actually means `drep.skey, drep.vkey`; when you unfold the brackets (as would bash do).

Reply:
Oh right, sorry. A typo got in where I was doing those changes.

Reply:
removed in: https://github.com/IntersectMBO/cardano-node/pull/6008